docs/whatsnew/3.0.rst

```diff
@@ -135,15 +136,32 @@ To get full documentation on the usage and available options do ``showtable
 
 .. _whatsnew-3.0-hdf5-mixins:
 
-HDF5 Table writer supports mix-in columns (e.g. time, coordinates, etc.)
-========================================================================
+Lossless writing and reading mixin columns to HDF5 and FITS
+===========================================================
 
+It is now possible to write Table :ref:`mixin_columns` such as
+:class:`~astropy.coordinates.SkyCoord`, :class:`~astropy.time.Time`, and
+:class:`~astropy.units.Quantity` (and subclasses like
+:class:`~astropy.coordinates.EarthLocation`) to both HDF5 and FITS with no loss of
+information.  All object attributes and metadata are stored to file and the object is
+exactly regenerated (to the bit level) when reading back into astropy.
+
+For details see the :ref:`fits_astropy_native` section.
 
 .. _whatsnew-3.0-fits-time-support:
 
 Support for FITS time standard
 ==============================
 
+The `FITS Time standard paper <http://adsabs.harvard.edu/abs/2015A%26A...574A..36R/>`_
+defines the formats and keywords used to represent timing information in FITS files. The
+Astropy FITS package now provides support for reading and writing native
+:class:`~astropy.time.Time` table columns using this format. This is described within the
+:ref:`FITS unified I/O interface <table_io_fits>` and examples of usage can be found in
+the :ref:`fits_astropy_native` section.
+
+Implementation of this feature was made possible through the support of the Google Summer
```

Comment:
Maybe link to the specific project page?  Or just give the name of the student to make it clear which project it was? (It was Aarya Patil, right?)